src/Akka.Logger.NLog/NLog.config

```diff
@@ -1,10 +0,0 @@
-﻿<?xml version="1.0" encoding="utf-8" ?>
```

Comment:
Why have you removed this file ?

Reply:
If my memory serves me right it is used to provide a sane default for usage of the logging framework, out of the box.

Reply:
Then it has to be included in the nuget-package, else it will not be available. Checked the nuget-packages (https://www.nuget.org/packages/Akka.Logger.NLog) and they do not copy this file (also dangerous to do so). But I can restore the file, doesn't matter to me. Just thought it was clutter.

Reply:
Yes please. But rename it to NLog.Example.Config. To better portray its function.

Reply:
Done